pkg/maps/policymap/policymap.go

```diff
@@ -47,6 +48,40 @@ const (
 	AllPorts = uint16(0)
 )
 
+type policyFlag uint8
+
+const (
+	policyFlagDeny = 1 << 0
+)
+
+type PolicyEntryFlags uint8
+
+// UInt8 returns the UInt8 representation of the ServiceFlags.
+func (pef PolicyEntryFlags) UInt8() uint8 {
+	return uint8(pef)
+}
+
+func (pef PolicyEntryFlags) is(pf policyFlag) bool {
+	return uint8(pef)&uint8(pf) != 0
+}
+
+func (pef PolicyEntryFlags) IsDeny() bool {
+	return pef.is(policyFlagDeny)
+}
+
+// String returns the string implementation of ServiceFlags.
+func (pef PolicyEntryFlags) String() string {
+	var str []string
+
+	if pef.IsDeny() {
+		str = append(str, "Deny")
+	} else {
+		str = append(str, "Allow")
+	}
+
+	return strings.Join(str, ", ")
+}
```

Comment:
I think this can be simplified to:
```suggestion
func (pef PolicyEntryFlags) String() string {
	if pef.IsDeny() {
		return "Deny"
	}
	return "Allow"
}
```